todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -61,7 +65,14 @@ const createTodoItem = createWidgetBase
 							state: { classes: [ 'toggle' ], checked }
 						}),
 						d(createLabel, {
-							listeners: { dblclick: () => { todoEdit.do(state); } },
+							listeners: {
+								dblclick: () => { todoEdit.do(state); },
+								keypress: (event: KeyboardEvent) => {
+									if (event.which === 13) {
```

Comment:
Yeah, it doesn't really matter for these. I was just thinking event.which would be better in the long run, so if we ever need to check for a letter key we won't have a mix of `which` and `keyCode`